### PR TITLE
Add Beta Testing fallback menu item

### PR DESCRIPTION
**Pull request by claudiucelfilip**
#### Changes proposed in this Pull Request

This adds the nav link for the Beta Testing page in the Fallback Menu. 
The page is added in PR #54346 and the nav link to it is added in D63456-code.

NOTE: #54346 and D63456-code should be deployed before merging this.

#### Testing instructions

* The Fallback Menu briefly appears before the response from the Public API enpoint /wpcom/v2/sites/{BLOG_ID}/admin-menu. You can block this URL to prevent the response from happening. You can also Clear Site Data to make sure the Fallback Menu is used.
* The Beta Testing link should appear as the last item, in the left Sidebar menu:
Screenshot on 2021-07-15 at 14-00-18

* The link should not appear on P2 sites
* It should open the /beta-testing page

### CODE REVIEW

File: client/my-sites/sidebar-unified/static-data/fallback-menu.js [hunk 1]
```diff
@@ -573,6 +574,13 @@ export default function buildFallbackResponse( {
 					},
 			  ]
 			: [] ),
+		{
+			icon: BETA_TESTING_ICON,
+			slug: 'beta-testing',
+			title: translate( 'Beta Testing' ),
```
Comment (a8ci18n) [inline client/my-sites/sidebar-unified/static-data/fallback-menu.js]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **122** times:
`translate( 'Beta Testing', { context: 'Plugin Installer'} )` *ES Score: 16*

**Review by danielwrobert — APPROVED** [review]
LGTM! 🚀 

Tested and confirmed menu item renders and links to the expected path with API endpoint blocked.

**Comment by claudiucelfilip** [comment]
@danielwrobert I added a condition so that the link doesn't appear on P2

**Review by danielwrobert — APPROVED** [review]
Re-tested with the latest updates.

LGTM! 🚀

**Pull request merged**